### PR TITLE
Add respawn anchor support.

### LINKED ISSUE #2329: Not respawning at respawn anchors
Server: Paper 1.16.1 build 44
Multiverse-Core: 4.1.0
Multiverse-Netherportals 4.1.0

I have a nether world.  I have a resource anchor in that nether world.  The anchor is charged.  
I set my respawn point using the anchor (and get the message, and the achievement).
I die. 
The resource anchor uses one of its charges.
I respawn at the world's default spawn point.   

I removed all of my plugins and added them in one by one.  I don't see this behaviour until MV is installed.

### PR DESCRIPTION
**Pull request by benwoo1110**
Fixes #2329

Uses reflection so that lower mc version compatibility is retained.

### CODE REVIEW

**Comment by benwoo1110** [comment]
**test build:** [Multiverse-Core-4.2.3-SNAPSHOT.jar.zip](https://github.com/Multiverse/Multiverse-Core/files/6027580/Multiverse-Core-4.2.3-SNAPSHOT.jar.zip)

File: src/main/java/com/onarandombox/MultiverseCore/utils/CompatibilityLayer.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        }
+        try {
+            return (boolean) checkAnchorSpawn.invoke(event);
+        } catch (Exception e) {
+            Logging.warning("Error checking for: %s", checkAnchorSpawn);
+            e.printStackTrace();
+        }
+        return false;
+    }
+}
```
Comment (dumptruckman) [inline src/main/java/com/onarandombox/MultiverseCore/utils/CompatibilityLayer.java:46]: nit: we should have a blank line at the end of files.

**Review by dumptruckman — APPROVED** [review]
I approve of the way you did this CompatiblityLayer.

File: src/main/java/com/onarandombox/MultiverseCore/utils/CompatibilityLayer.java (hunk 1 above)
Comment (benwoo1110) [reply src/main/java/com/onarandombox/MultiverseCore/utils/CompatibilityLayer.java:46]: ah always forget that

**Comment by VoidsSpawn** [comment]
Is there any fix for this? for 1.19? (the specific issue im experiencing is on a many-mv instance manhunt server)

**Pull request merged**